args.js

```diff
@@ -26,6 +26,10 @@ module.exports = yargs
         describe: 'Upload challenge files with a Jekyll-compatible YAML front matter (see https://jekyllrb.com/docs/frontmatter)',
         type: 'boolean',
         default: false
+    }).option('force-renewal', {
+        describe: 'Force renewal of certificate, even if it expires in more than 30 days',
```

Comment:
This should be phrased with plural instead:

> Force renewal of certificates, even if they expire in more than 30 days.

Reply:
Two issues with that:
1. `--production` also is in singular:

    > Obtain a real certificate instead of a dummy one
    
2. Technically you only obtain one certificate per execution, right?

Should I still change the text?